src/element/resizeElements.ts

```diff
@@ -186,13 +186,13 @@ const validateTwoPointElementNormalized = (
 };
 
 const getPerfectElementSizeWithRotation = (
-  elementType: string,
+  activeTool: string,
```

Comment:
this is actually not a param for activeTool as we're passing excaldiraw elements, so `elementType` naming was correct